Arduino/victoria_teensy_ros_node/victoria_teensy_ros_node.ino

```diff
@@ -235,25 +239,25 @@ void setup() {
   last_encoder_read_time = current_time;
 
   // Initialize ros debug publishers
-#ifdef ENABLE_BUMPER_DEBUG
-  ros_nh.advertise(ros_bumper_debug_pub);
-  timer.every(500, doBumperDebug);
-#endif
-#ifdef ENABLE_ENCODER_DEBUG
-  ros_nh.advertise(ros_encoder_debug_pub);
-  timer.every(500, doEncoderDebug);
-#endif
-#ifdef ENABLE_IMU_DEBUG
-  ros_nh.advertise(ros_imu_debug_pub);
-  timer.every(500, doImuDebug);
-#endif
-#ifdef ENABLE_MAGNETOMETER_DEBUG
-  ros_nh.advertise(ros_magnetometer_debug_pub);
-  timer.every(500, doMagnetometerDebug);
-#endif
-#ifdef ENABLE_TEENSY_DEBUG
-  ros_nh.advertise(ros_teensy_debug_pub);
-#endif
+  if (publish_bumper_debug_info_freq_hz != 0) {
```

Comment:
And some of the others are going to be going away in the near future with custom messages.